README cleanup and simplification (#3104)
hashgraph/hedera-mirror-node: README.md

by receiving pre-constructed files from the network, validating those, populating a database, and providing APIs to
expose the data. This approach provides the following advantages:

- Lower compute and bandwidth requirements
- It allows users to only save what they care about, and discard what they don’t (lower storage requirement)
- Easy searchable database so the users can add value quickly
- Easy to consume APIs to make integrations faster

## Architecture

### Main Nodes

- When a transaction reaches consensus, Hedera nodes add the transaction and its associated record to a
  [record file](https://docs.hedera.com/guides/docs/record-and-event-stream-file-formats#version-5-record-stream-file-format)
  .
- Record files contain the hash of the previous record file, thus creating an unbreakable validation chain.
- The file is closed on a regular cadence and a signature file is generated by the node for the record file.
- The record and signature files from each node are then uploaded to Amazon S3 and Google Cloud Storage.

### Mirror Nodes

- This mirror node software downloads signature files from cloud storage.

> - The signature files are validated to ensure at least 1/3 of the nodes in the address book (stored in a `0.0.102` file)

Author: Updated

  have the same signature.

> - For each valid signature file, the corresponding record file is then downloaded from the cloud and its hash verified

Reviewer: ```suggestion
- For each valid signature file, the corresponding record file is then downloaded from cloud storage and its hash verified
```

just to be consistent with line 45.

Author: Updated

additional business logic to support applications built using the Hedera network.

While mirror nodes receive information from the main nodes, they do not contribute to consensus on the network, and
their votes are not counted. Only the votes from the main nodes are counted for determining consensus. The trust of the
Hedera network is derived based on the consensus reached by the main nodes. That trust is transferred to the mirror
nodes using cryptographic signatures on a chain of files.

Eventually, the mirror nodes will be able to run the same code as the Hedera nodes so that they can see the transactions
in real time. To make the initial deployments easier, the mirror node strives to take away the burden of running a full
Hedera node through creation of periodic files that contain processed information (such as account balances or
transaction records), and have the full trust of the main nodes. The mirror node software reduces the processing burden
by receiving pre-constructed files from the network, validating those, populating a database, and providing APIs to
expose the data. This approach provides the following advantages:

- Lower compute and bandwidth requirements

> - It allows users to only save what they care about, and discard what they don’t (lower storage requirement)

Reviewer 2: ```suggestion
- Allows users to only save the data that they care about (lower storage requirement)
```

Author: Updated

to understanding the mirror node operations.

- [Git](https://git-scm.com/about)
- [Go](https://golang.org/)
- [Docker](https://docs.docker.com/engine/reference/commandline/docker/)
- [Helm](https://helm.sh)
- [Maven](https://maven.apache.org/guides/getting-started/index.html)

> - [NodeJS](https://nodejs.org/en/about/)

Reviewer: nit: make it consistent with what's on the official about page
```suggestion
- [Node.js](https://nodejs.org/en/about/)
```

Author: Updated

to understanding the mirror node operations.

- [Git](https://git-scm.com/about)
- [Go](https://golang.org/)
- [Docker](https://docs.docker.com/engine/reference/commandline/docker/)
- [Helm](https://helm.sh)
- [Maven](https://maven.apache.org/guides/getting-started/index.html)
- [Node.js](https://nodejs.org/en/about/)
- [PostgreSQL](https://www.postgresql.org/docs)
- [Spring Boot](https://docs.spring.io/spring-boot/docs/current/reference/html/getting-started.html#getting-started)

### Prerequisite Tools

Ensure these tools are installed (note minimum versions) prior to running the mirror node:

- [Docker Desktop 4.3+](https://www.docker.com/products/docker-desktop)

### Running

For production use, we recommend using Kubernetes and to deploy using our Helm [chart](charts). Hedera managed mirror

> nodes use Kubernetes and Helm for its deployments and is considered the most production-ready. As an alternative for

Reviewer 2: ```suggestion
nodes use Kubernetes and Helm for their deployments, and this process is considered the most production-ready. As an alternative for
```

Author: Updated
